amoro-ams/pom.xml

```diff
@@ -394,6 +394,12 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>com.github.pagehelper</groupId>
+            <artifactId>pagehelper</artifactId>
+            <version>${pagehelper.version}</version>
+        </dependency>
```

Comment:
github.pagehelper is released under MIT License. 

Will this have an impact on our graduation from the Apache incubation? @zhoujinsong @czy006

Reply:
The project is here https://github.com/pagehelper/Mybatis-PageHelper